src/components/newUserForm/NewUserForm.js

```diff
@@ -1,9 +1,57 @@
 import React from 'react';
 
-export const NewUserForm = () => {
+export const NewUserForm = ({
+  setFirstname,
+  setLastname,
+  setUsername,
+  usernameCheck,
+  setPassword,
+  setConfirm,
+  passwordCheck
+  }) => {
   return (
-    <div style={{border: "1px solid black"}}>
-      <h3>New Form Component</h3>
+    <div>
+      Add Patient Account<br />
+      <br/>
+      <label>First Name</label><br/>
```

Comment:
The `input`s need to be a child of the `label` for semantic correctness. If you want to keep them separate then you'll need to link the `label`s to the `input`s.